app/posts/apply-for-teacher-training/2020-02-20-declaring-safeguarding-concerns.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Declaring any safeguarding issues
+description: Starting a conversation between providers and candidates about safeguarding issues.
+date: 2020-02-20
+---
+
+Teacher training providers need to recruit candidates who are safe to work with children and young people. They use various sources of evidence to make a judgement on this.
+
+However, providers need to ask candidates about potential safeguarding issues (such as criminal convictions) in the application form. Encouraging candidates to disclose potential issues at this early stage is essential so that they can:
+
+* give advice about whether it's worth pursuing an application (saving time and money, reducing safeguarding risks during interview and helping to avoid damage to their reputation)
+* build trust with candidates who do have a conviction or caution but can still teach
+
+Our [initial design for this section of the application form](/apply-for-teacher-training/suitability-to-work-with-children) aimed to meet these provider needs.
+
+However, we also need to consider how candidates respond to this design.
+
+## User needs
+
+### Provider need
+
+As a provider
+
+I need to know about a candidate's past
+
+So that I can make a judgement on whether to pursue their application
+
+### Candidate need
+
+As a candidate
+
+I need to feel comfortable to disclose information about my past
+
+So that I know whether it's worth carrying on with my application
+
+## Hypothesis
+
+### Context
+
+We carried out user research with candidates on the [initial design](/apply-for-teacher-training/suitability-to-work-with-children/02-your-suitability-to-work-with-children.png).
+
+Our research was limited due to the difficulty of recruiting candidates who have convictions or cautions, who have been banned from working with children or who have been removed from teacher training in the past due to unprofessional behaviour.
+
+However, we were able to observe a few pain points for candidates. The content lacks sufficient clarity, particularly around:
+
+* what sort of information to declare
+* what the purpose of this section is
+* whether they should be open or not  
+
+### Our hypothesis
+
+1. If we list the types of checks providers carry out
+Then candidates will know whether to share information or not
+Because they'll know what type of information providers are looking for
+
```

Comment:
Why the double line breaks between hypotheses?

Reply:
oh